### DISCUSSION TITLE
Being notitfied of new content in the Foundations book

### DISCUSSION THREAD

**Discussion by phil-blain** [ProjectPythia/projectpythia.github.io#239] [General]
Hi!

I just finished the Pythia foundation book and learned a lot of cool stuff, congrats to the team!

I saw that several sections of the book are listed as "in construction", with more tutorial mentioned as being planned. It would be really great if we could subscribed somehow to be notified of such new content. 

A suggestion would be to create GitHub "Releases" for the https://github.com/ProjectPythia/pythia-foundations repo whenever a new tutorial is added (since on GitHub you can subscribe to be notified of new releases in a specific repository).

**Comment by brian-rose** [comment]
I like the idea of creating tagged releases for the Foundations source.

Related, we're trying to track the use of those "under construction" headers over at https://github.com/ProjectPythia/pythia-foundations/issues/231
